src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -93,9 +102,15 @@ private static SegmentedList<VSCompletionItem> SortCompletionItems(AsyncCompleti
                     data = new AsyncCompletionSessionDataSnapshot(sessionData.CombinedSortedList, data.Snapshot, data.Trigger, data.InitialTrigger, data.SelectedFilters,
                         data.IsSoftSelected, data.DisplaySuggestionItem, data.Defaults);
                 }
-                else if (sessionData.ExpandedItemsTask != null)
+                else if (!ShouldHideExpandedItems() && sessionData.ExpandedItemsTask is not null)
```

Comment:
instead of `!ShouldHide` consider `ShouldShow`

Reply:
done